codegenerator/model/api.go

```diff
@@ -177,14 +170,14 @@ type ` + api.apiDef.Name + ` tcclient.Client
 	}
 
 	content += "//  if err != nil {\n"
-	content += "//  	// handle errors...\n"
+	content += "//          // handle errors...\n"
 	content += "//  }"
 	content += `
 func New(credentials *tcclient.Credentials) *` + api.apiDef.Name + ` {
 	return &` + api.apiDef.Name + `{
 		Credentials: credentials,
-		BaseURL: DefaultBaseURL,
-		Authenticate: credentials != nil,
+		Service: "` + api.ServiceName + `",
+		Version: "` + "v1" + `",
```

Comment:
@jhford has added this to the API reference as `api.apiVersion` (defaulting to `"v1"` if not present)

Reply:
Ah great, the `"v1"` was a stop gap until we had somewhere to get this data from, I'll update it!